.github/workflows/ci.yml

```diff
@@ -39,10 +39,12 @@ jobs:
         - { name: Linux Clang,                    os: ubuntu-22.04, flags: -DCMAKE_CXX_COMPILER=clang++ -DSFML_RUN_DISPLAY_TESTS=ON -GNinja , gcovr_options: '--gcov-executable="llvm-cov-$CLANG_VERSION gcov"' }
         - { name: Linux GCC DRM,                  os: ubuntu-22.04, flags: -DSFML_USE_DRM=ON -DSFML_RUN_DISPLAY_TESTS=OFF -GNinja }
         - { name: Linux GCC OpenGL ES,            os: ubuntu-22.04, flags: -DSFML_OPENGL_ES=ON -DSFML_RUN_DISPLAY_TESTS=ON -GNinja }
-        - { name: macOS,                          os: macos-12, flags: -GNinja }
-        - { name: macOS Xcode,                    os: macos-12, flags: -GXcode }
```

Comment:
It's already questionable whether having a macOS Ninja job and macOS Xcode job is useful. I think for these ARM builds we don't need to duplicate them for Xcode. That'll save us some CI time.

Reply:
It's hard to write CMake code that works with one generator on a given OS but not another generator.

> not that CI time is really a useful or strong argument here.

What do you mean? We've had _many_ PRs spent optimizing CI runtime. The fewer jobs we have the faster CI completes.